src/renderer/pages/Wallet/WalletAll.vue

```diff
@@ -304,6 +365,21 @@ export default {
 
     showVotedDelegates () {
       return some(this.selectableWallets, wallet => wallet.hasOwnProperty('votedDelegate'))
+    },
+
+    selectableWalletsWithPlaceholders () {
+      const placeholder = {
+        address: this.$t('COMMON.WALLET'),
+        balance: 0,
+        isPlaceholder: true
+      }
+
+      // Add two placeholders
+      return [
```

Comment:
@luciorubeens I'm seeing those 2 placeholders, but I can't figure out the reason

Reply:
@j-a-m-l I did according to the design, and in the task it is written that this is necessary 🤷‍♂️

Reply:
But in that design it displays as many wallets as necessary to fill the screen.

Anyway, I think that it would confuse users.